ipatests/test_integration/test_ipahealthcheck.py

```diff
@@ -507,6 +507,45 @@ def test_ipahealthcheck_dogtag_ca_connectivity_check(self, restart_tomcat):
             assert additional_msg in check["kw"]["msg"] or \
                    additional_msg == check["kw"]["error"]
 
+    def test_ipahealthcheck_ca_not_configured(self):
+        """
+        Test if the healthcheck ignores pki-tomcat errors
+        when CA is not configured on the machine
+        Related: https://github.com/freeipa/freeipa-healthcheck/issues/201
+        """
+        # uninstall replica installed by class' install method
+        tasks.uninstall_replica(self.master, self.replicas[0])
+
+        # install it again without CA
+        tasks.install_replica(self.master,
+                              self.replicas[0],
+                              setup_ca=False,
+                              setup_dns=True,
+                              extra_args=['--no-dnssec-validation']
+                              )
+
+        # Init a user on replica to assign a DNA range
+        tasks.kinit_admin(self.replicas[0])
+        tasks.user_add(
+            self.replicas[0], 'ipauser1', first='Test', last='User',
+        )
+
+        returncode, data = run_healthcheck(self.replicas[0],
+                                           failures_only=True)
+        assert returncode == 0
+        assert len(data) == 0
+
+        # restore the replica original configuration
+        tasks.user_del(self.replicas[0], 'ipauser1')
+        tasks.uninstall_replica(self.master, self.replicas[0])
+        tasks.install_replica(
+            self.master,
+            self.replicas[0],
+            setup_dns=True,
+            extra_args=['--no-dnssec-validation']
+        )
+
```

Comment:
You may be able to move this test to last so you don't have to reinstall the replica. You'd need a big note to explain that no test that relies on a replica should be added after it.
Otherwise LGTM.